cmd/juju/commands/ssh_container.go

```diff
@@ -63,7 +65,9 @@ type ModelAPI interface {
 }
 
 // SetFlags sets up options and flags for the command.
-func (c *sshContainer) SetFlags(f *gnuflag.FlagSet) {}
+func (c *sshContainer) SetFlags(f *gnuflag.FlagSet) {
+	f.BoolVar(&c.remote, "remote", false, "Target on the workload or operator pod (k8s-only)")
```

Comment:
Elsewhere we use "operator" as the arg name, so should be consistent and use that here too.

f.BoolVar(&c.operator, "operator", false, "ssh into the operator (k8s-only)")

Reply:
For action stuff, we expect to target workload pod(operator==false) by default.
Here is the opposite, we expect to target operators for debug-hooks, etc.

Reply:
as discussed, we decide to use `remote` here.